ah_bootstrap.py

```diff
@@ -210,6 +212,8 @@ def main(cls, argv=None):
         auto_use = config.pop('auto_use', False)
         bootstrapper = cls(**config)
 
+        _builtins._astropy_helpers_config = config
```

Comment:
I didn't even think about trying to support these options.  But yeah, let's not do this.

I've been thinking of getting rid of those altogether.  They can already be edited in setup.cfg if need be.  Furthermore, I've been thinking of adding support for a "site.cfg" which, if present, could override settings in setup.cfg.  That way developers and downstream packagers who need to make tweaks can put them there instead.